include/JSystem/JKernel/JKRAramHeap/JKRAramHeap.h

```diff
@@ -10,33 +10,38 @@ class JKRAramHeap : public JKRDisposer {
     enum EAllocMode {
         HEAD = 0,
         TAIL = 1,
+
+        __EALLOCMODE_FORCE_ENUM_S32 = INT32_MAX,
+        __EALLOCMODE_FORCE_ENUM_SIGNED = -1,
     };
 
 public:
     JKRAramHeap(u32, u32);
-    virtual ~JKRAramHeap(void);
-
-    void alloc(u32, EAllocMode);
-    void allocFromHead(u32);
-    void allocFromTail(u32);
-    void getFreeSize(void);
-    void getTotalFreeSize(void);
+    virtual ~JKRAramHeap();
+
+    JKRAramBlock* alloc(u32, EAllocMode);
+    JKRAramBlock* allocFromHead(u32);
+    JKRAramBlock* allocFromTail(u32);
+    u32 getFreeSize(void);
+    u32 getTotalFreeSize(void);
+    // u32 getUsedSize(void);
     void dump(void);
 
     u8 getCurrentGroupID() { return this->mGroupId; }
 
     JKRHeap* getMgrHeap() { return this->mHeap; }
 
+private:
     void lock() { OSLockMutex(&this->mMutex); }
```

Comment:
```suggestion
    void lock() { OSLockMutex(&mMutex); }
```
Usage of `this` for member access is redundant, goes for the other instances as well.